pages/1.13/cli/autocompletion/index.md

```diff
@@ -7,29 +7,27 @@ excerpt: Enabling autocompletion for the CLI
 enterprise: false
 ---
 
-Autocompletion for the CLI is supported for `bash` and `zsh`.
-It was released with the 1.12 release so the CLI must be of version 0.7.x or greater. Please see https://downloads.dcos.io/cli/index.html for a correct version. \
-You will also need to have the `dcos` executable in your `$PATH`.
+Autocompletion for the CLI is supported for `bash` and `zsh`. You will need to have the `dcos` executable in your `$PATH` and use it with a cluster running on DC/OS 1.12 or later.
 
-Autocompletion for the CLI will only complete commands that are available at the time of completion. If you are not attached to any cluster, only the following commands will be able to complete: `auth`, `cluster`, `config`, `help` and `plugin`. To complete other commands you will have to be attached to a cluster and added the respective plugins.\
-Completions are available for the CORE CLI plugin and the EE CLI plugin.
+Autocompletion for the CLI will only complete commands that are available at the time of completion. If you are not attached to any cluster, only the following commands will be able to complete: `auth`, `cluster`, `config`, `help` and `plugin`. To complete other commands, you will have to be attached to a cluster and added the respective plugins.\
```

Comment:
I don't understand what this means 
Autocompletion for the CLI will only complete commands that are available at the time of completion